lensless/drunet/basicblock.py

```diff
@@ -0,0 +1,788 @@
+# #############################################################################
+# basicblock.py
+# =================
+# Original Repo:
+# https://github.com/cszn/DPIR
```

Comment:
Perhaps you could put link to exact file (with commit iD): https://github.com/cszn/DPIR/blob/15bca3fcc1f3cc51a1f99ccf027691e278c19354/models/basicblock.py